.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/acebook-rails-keeping-it-rails.iml" filepath="$PROJECT_DIR$/.idea/acebook-rails-keeping-it-rails.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
Not sure what this is? please explain

Reply:
files auto-generated by daniel's jetbrain IDE, will remove as he is now using atom